packages/account/src/Components/trading-assessment/trading-assessment-new-user.tsx

```diff
@@ -70,7 +70,7 @@ const TradingAssessmentNewUser = observer(
                 disabled_items={disabled_items}
                 should_move_to_next={false}
                 is_independent_section={false}
-                is_mobile={is_mobile}
+                is_mobile={!isDesktop}
```

Comment:
since we are passing !isDesktop as the value, lets change the name is_mobile to is_responsive?